src/annotated_lcf.fun

```diff
@@ -0,0 +1,40 @@
+functor AnnotatedLcf
```

Comment:
That's interesting about item `2`. I'm guessing that's basically what was going on with the problem I encountered with the `where` clause in the other thread?